tofu/main/aws/terraform.tf

```diff
@@ -1,5 +1,5 @@
 terraform {
-  required_version = "1.8.2"
+  required_version = ">= 1.8.1"
```

Comment:
why the downgrade?

Reply:
This should be a more accurate version constraint based on the opentofu features used